snippets/SMTP/smtp_example.cs

```diff
@@ -0,0 +1,32 @@
+using System;
+using System.Net;
+using System.Net.Mail;
+ 
+public class EmailTest
+{
+   public static void Main()
+   {
+        MailAddress to = new MailAddress("developers@sparkpost.com");
```

Comment:
let's not have them send to developers@sparkpost.com - the cURL examples use `<REPLACE_WITH_YOUR_EMAIL_ADDRESS>`